pkg/integration_test/pdctl_test.go

```diff
@@ -734,6 +734,145 @@ func (s *integrationTestSuite) TestConfig(c *C) {
+	args = []string{"-u", pdAddr, "operator", "remove", "1"}
+	_, _, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+
+	// operator add transfer-region <region_id> <to_store_id>...
+	args = []string{"-u", pdAddr, "operator", "add", "transfer-region", "1", "2", "3"}
+	_, _, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+	args = []string{"-u", pdAddr, "operator", "show", "region"}
+	_, output, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+	c.Assert(strings.Contains(string(output), "remove peer on store 1"), IsTrue)
+	args = []string{"-u", pdAddr, "operator", "remove", "1"}
+	_, _, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+
+	// operator add transfer-peer <region_id> <from_store_id> <to_store_id>
+	args = []string{"-u", pdAddr, "operator", "add", "transfer-peer", "1", "2", "3"}
+	_, _, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+	args = []string{"-u", pdAddr, "operator", "show"}
+	_, output, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+	c.Assert(strings.Contains(string(output), "remove peer on store 2"), IsTrue)
+	args = []string{"-u", pdAddr, "operator", "remove", "1"}
+	_, _, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+
+	// operator add merge-region <source_region_id> <target_region_id>
+	args = []string{"-u", pdAddr, "operator", "add", "merge-region", "1", "3"}
+	_, _, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+	args = []string{"-u", pdAddr, "operator", "show"}
+	_, output, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+	c.Assert(strings.Contains(string(output), "merge region 1 into region 3"), IsTrue)
+	args = []string{"-u", pdAddr, "operator", "remove", "1"}
+	_, _, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+	args = []string{"-u", pdAddr, "operator", "remove", "3"}
+	_, _, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+
+	// operator add split-region <region_id> [--policy=scan|approximate]
+	args = []string{"-u", pdAddr, "operator", "add", "split-region", "3", "--policy=scan"}
+	_, _, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+	args = []string{"-u", pdAddr, "operator", "show"}
+	_, output, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+	c.Assert(strings.Contains(string(output), "split region with policy SCAN"), IsTrue)
+	args = []string{"-u", pdAddr, "operator", "remove", "3"}
+	_, _, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+
+	// operator add split-region <region_id> [--policy=scan|approximate]
+	args = []string{"-u", pdAddr, "operator", "add", "split-region", "3", "--policy=approximate"}
+	_, _, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+	args = []string{"-u", pdAddr, "operator", "show"}
+	_, output, err = executeCommandC(cmd, args...)
+	c.Assert(err, IsNil)
+	c.Assert(strings.Contains(string(output), "split region with policy APPROXIMATE"), IsTrue)
```

Comment:
The test processes are all the same. I think you can make it table-driven:
`cmds := []struct{args []string, outputPattern string} {...}`